ddl.dic

```diff
@@ -1022,6 +1050,75 @@ save_enumeration_default.value
 
 save_
 
+save_ENUMERATION_DEFAULTS
+
+    _definition.id                ENUMERATION_DEFAULTS
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-06-01
+    _description.text
+;
+    Loop of pre-determined default enumeration values indexed to a
+    data item by the (potentially) multiple keys in the item
+    _enumeration.def_index_ids.
+;
+    _name.category_id             ENUMERATION
+    _name.object_id               ENUMERATION_DEFAULTS
+    _category_key.name            '_enumeration_defaults.index'
+
+save_
+
+save_enumeration_defaults.index
+
+    _definition.id                '_enumeration_defaults.index'
+    _definition.class             Attribute
+    _definition.update            2023-06-01
+    _description.text
+;
+    Index key in the list default values referenced to by the value of
+    _enumeration.def_index_ids. The values given must be in the same order as
+    specified by _enumeration.def_index_ids.
+;
+    _name.category_id             enumeration_defaults
+    _name.object_id               index
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Array
+    _type.dimension               '[]'
+    _type.contents                Code
+    _method.purpose               Definition
+    _method.expression
+;
+    _enumeration_defaults.index = [ _enumeration_default.index ]
+;
+
+save_
+
+save_enumeration_defaults.value
+
+    _definition.id                '_enumeration_defaults.value'
+    _definition.class             Attribute
+    _definition.update            2023-06-01
+    _description.text
+;
+    Default enumeration value in the list referenced by the value of
+    _enumeration.def_index_ids. The reference index key is given by
+    the value of _enumeration_defaults.index.
+;
+    _name.category_id             enumeration_defaults
+    _name.object_id               value
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Implied
+    _type.contents                Implied
+    _method.purpose               Definition
+    _method.expression
+;
+    _enumeration_defaults.value = _enumeration_default.value
```

Comment:
This dREL might almost work if it read:
```
_enumeration_defaults.value = _enumeration_default.value[_enumeration_defaults.index[0]]
```
except for the fact that `index` could be a mulit-entry list, in which case who knows if the first entry is the one you want to use. So I'd drop this as well.